R/utilities-figures.R

```diff
@@ -486,10 +486,14 @@ createPlotsFromExcel <- function(
 
   # The values can be enclosed in "" in case the title should contain a ','.
   # Split the input string by ',' but do not split within ""
-  dfPlotGrids$plotIDs <- unlist(trimws(scan(
-    text = as.character(dfPlotGrids$plotIDs), what = "character", sep = ",",
-    quiet = TRUE
-  )))
+  # Have to do it one row at a time, otherwise it returns one separate list entry
+  # for each plot it (and not lists of plot ids)
+  dfPlotGrids$plotIDs <- lapply(dfPlotGrids$plotIDs, \(plotId){
```

Comment:
@Felixmil this is not related to adding ontogenies, but a required bugfix... sorry for mixing it up in the same PR.

Reply:
No problem, but why is there `\(plotId)`. Is it like an anonymous function ?

Reply:
it's like `function(x)`, was introduced with R4.1

https://towardsdatascience.com/the-new-pipe-and-anonymous-function-syntax-in-r-54d98861014c

Reply:
Totally missed this when 4.1 was released! Anyway, not a big fan of this kind of shortcut, I think it makes the code less clear.